mysqld_exporter.go

```diff
@@ -56,10 +58,6 @@ var (
 		"collect.info_schema.innodb_metrics", false,
 		"Collect metrics from information_schema.innodb_metrics",
 	)
-	collectGlobalStatus = flag.Bool(
```

Comment:
There's different opinions on whether that should even be like that there, but I'd see these flags here as different. They are more about enabling/disabling a collector completely rather than doing some detailed internal config of it. That is also done from main in the node exporter.